packages/transport-bridge/src/core.ts

```diff
@@ -104,11 +112,15 @@ export const createApi = (apiArg: 'usb' | 'udp' | AbstractApi, logger?: Log) =>
         }
     };
 
-    const enumerate = async () => {
+    const enumerate = async ({ signal }: { signal: AbortSignal }) => {
         await sessionsClient.enumerateIntent();
 
-        const enumerateResult = await api.enumerate();
+        const enumerateResult = await api.enumerate(signal);
         if (!enumerateResult.success) {
+            await sessionsClient.enumerateDone({
+                descriptors: [],
+            });
+
```

Comment:
should be possible to remove lines 116 and 120-123. 
concurrent enumeration has now been allowed for some time and only now I am removing some left over redundant code in https://github.com/trezor/trezor-suite/pull/12954

Reply:
Yes, it seems to work. So I suggest merging #21954 first and dropping [3861988](https://github.com/trezor/trezor-suite/pull/12769/commits/3861988743306fd275fb3521a9e6e61af9bfe7af)

Reply:
well, not dropping it all together actually, there are imho valuable changes here. may I force push this commit?